roadmaps/2025roadmap.md

```diff
@@ -0,0 +1,47 @@
+# OpenMDAO Development Map 2023
+
+Author: Rob Falck
+Date: 2025-05-12
+
+## What is this roadmap for?
+As in previous years, this roadmap exists to provide general guidance for our development decisions in the coming year, and to provide a retrospective for our performance in the previous year.
+
+## Performance Assessment for 2024
+
+While we neglected to post an official roadmap for 2024, we made significant progress on reducing the activation energy for users.
+
+We made considerable strides into automatic differentiation by closely coupling the capability of the JAX framework with OpenMDAO's JAXExplicitComponent and JaxImplicitComponent. This is in addition to existing AD capability on the Julia side via `OpenMDAO.jl` and the work being developed on the Computational Systems Design Language (CSDL) at UCSD.
+
+AnalysisDriver is a generalization of capability regarding model analysis and visualization. DOEDriver's behavior was governed by the optimization problem as defined upon the system. Performing a simple parameter sweep across model inputs wasn't simple to do. The work on the AnalaysisDriver and related visualizations should improve the situation.
```

Comment:
"generalization of capability" --> "generalization of DOEDriver's capability"  ??